fluentcheck/classes/is_cls.py

```diff
@@ -14,6 +14,4 @@
 
 class Is(__IsBool, __IsCollections, __IsDicts, __IsEmptiness, __IsGeo,
          __IsNumbers, __IsSequences, __IsStrings, __IsTypes, __IsUUIDs):
-
-    def __init__(self, object_under_test: Any):
-        super().__init__(object_under_test)
+    pass
```

Comment:
Can we omit the constructor entirely rather than leaving a pass here?

Reply:
It totally depends on you. I can remove the pass but I'd suggest keeping it to avoid any compilation errors.